cdap-app-fabric/src/main/java/co/cask/cdap/app/guice/AppFabricServiceRuntimeModule.java

```diff
@@ -143,7 +144,7 @@ protected void configure() {
                                mapBinder.addBinding(Constants.Service.METRICS)
                                         .to(InMemoryMetricsServiceManager.class);
                                mapBinder.addBinding(Constants.Service.APP_FABRIC_HTTP)
-                                        .to(AppFabricServiceManager.class);
+                                        .to(InMemoryAppFabricServiceManager.class);
```

Comment:
may be we can reuse this map, as standalone and in-memory both use the same map entries.